lib/go/templates/cmd/manifest/manifest.go

```diff
@@ -829,7 +829,7 @@ func generateManifest(env templates.Environment) *manifest {
 	))
 
 	m.addTemplate(generateTemplate(
-		"SCO.17", "Register Node with PoP",
+		"SCO.17", "Register Node",
```

Comment:
A wallet would use `SCO.03` or `SCO.17` but not both at the same time. `SCO.17` is just the new "Register Node" transaction.